### PR TITLE
pandas task

### PR DESCRIPTION
**Pull request by asifnaseebi**

### CODE REVIEW

File: task/t2_pandas.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+
+FILE_PATH = "f1.csv"
+data_frame = pd.read_csv(FILE_PATH)
+
+MAX_TEMP_COL = 'Max TemperatureC'
+MIN_TEMP_COL = 'Min TemperatureC'
```
Comment (Hamma111) [inline task/t2_pandas.py]: move constants to a constants.py file and import these variables from there

File: task/t3_pandas.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import pandas as pd
+
+FILE_PATH = "f2.csv"
+data_frame = pd.read_csv(FILE_PATH)
+
+dates_events = data_frame[data_frame[' Events'].isin(['Snow', 'Rain', 'Rain-Snow'])]['PKT'].tolist()
```
Comment (Hamma111) [inline task/t3_pandas.py]: why not move `' Events'` to a constant as well?

**Review by Hamma111 — APPROVED** [review]

File: task/t2_pandas.py (hunk 1 above)
Comment (asifnaseebi) [reply task/t2_pandas.py]: sir ap kiya kehna cha rhy hn samj nhi i

**Review by Hamma111 — APPROVED** [review]

**Comment by asifnaseebi** [comment]
pandas_task_2_march.

**Comment by asifnaseebi** [comment]
done

**Pull request merged**